app/javascript/components/rules/RuleEditorHeader.vue

```diff
@@ -30,6 +30,7 @@
         :idPrefix="'duplicate'"
         :forDuplicate="true"
         :selectedRuleId="rule.id"
+        @ruleSelected="handleRuleSelected($event)"
```

Comment:
no need to split up the logic into a helper function if you just use:
`@ruleSelected="$emit('ruleSelected', $event.id)"`